backend/app/api/journal.py

```
@@ -105,6 +113,179 @@ async def parse_stt(
+        exif_hints.append(extract_exif(data))
+
+    try:
+        result = await parse_photos_internal(
+            images=images,
+            exif_hints=exif_hints,
+            field_name=field_name,
+            crop=crop,
+            db=db,
+        )
+    except httpx.TimeoutException:
+        raise HTTPException(504, "Vision 분석 시간이 초과되었습니다.")
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)